internal/resources/policy/kind/common/target_kubernetes_resources_flatten_test.go

```diff
@@ -3,7 +3,7 @@ Copyright © 2022 VMware, Inc. All Rights Reserved.
 SPDX-License-Identifier: MPL-2.0
 */
 
-package recipe
+package common
```

Comment:
With this change(package name & file path) you may have to change all its references too, is there a specific reason for doing this change?

Reply:
This was change so I could be able to reuse **TargetKubernetesResourcesSchema**.

Reply:
It should be fine as long as the dependant resources are updated with this change